task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -15,6 +15,8 @@ spec:
       description: Tekton task test output.
     - name: IMAGES_PROCESSED
       description: Images processed in the task.
+    - name: RESULT_URI
```

Comment:
Do we need to update the ADR with this new result? 

https://github.com/konflux-ci/architecture/blob/main/ADR/0030-tekton-results-naming-convention.md

Reply:
No. Good idea, but let's go in the opposite direction and rename this to be something _non_standard, like `CLAMAV_RESULT_URI`. Something just for consumers who know _this_ task.

Reply:
Done in ef92eeb0c595fafba4b480fedeef861e239c10ef.

Reply:
Do we need a new result? if the scan results are `attached` to the container image they can be discovered using `oras discover ...`

Reply:
I do see a value in introducing a new result if it's going to be displayed in the UI (like we show the pull reference to the image). With that being said I still think we should use `oras attach`.

Reply:
Ack, let's do both.

Reply:
Tested at https://github.com/ralphbean/python-app/pull/51

```
❯ oras discover --distribution-spec v1.1-referrers-tag quay.io/redhat-user-workloads/ralphjbean-tenant/demo/python-app@sha256:5e3aff51d8342e2dbbfbabea4b647e580a022b471b2fc428f69931e7bb811017
quay.io/redhat-user-workloads/ralphjbean-tenant/demo/python-app@sha256:5e3aff51d8342e2dbbfbabea4b647e580a022b471b2fc428f69931e7bb811017
├── application/sarif+json
│   └── sha256:e55bc2e1bfc7be5339e95c332f6dc4aeefe8dd24d4009d252fa003bcb0dd23ad
└── application/vnd.clamav
    └── sha256:73ef70da4321875976f49e31f9a82607d8baef6cc4542dee7639390d4b98d726
```